examples/python/external_data_loader/main.py

```diff
@@ -14,6 +14,9 @@
 # It is up to you whether you make use of that shared recording ID or not.
 # If you use it, the data will end up in the same recording as all other plugins interested in
 # that file, otherwise you can just create a dedicated recording for it. Or both.
+#
+# Check out `re_data_source::RecommendedLoadSettings` documentation for an exhaustive listing of
```

Comment:
xxx

Reply:
ah that would be nice, linking python users to rust documentation hmmmm

Reply:
![image](https://github.com/rerun-io/rerun/assets/2910679/cc2e76e1-e23f-4de8-86dd-3a4b2ff63421)